src/cli/customer.rs

```diff
@@ -49,8 +49,8 @@ pub struct List {
 #[derive(Debug, StructOpt)]
 #[non_exhaustive]
 pub struct Show {
-    /// The channel ID, or a unique prefix of it.
-    pub prefix: String,
+    /// The channel label.
+    pub label: ChannelName,
```

Comment:
Did this really change? (I mean in functionality), can we still do a prefix instead of full label?

Reply:
Previously, this wanted a prefix of the channel ID, the long hex string, rather than the human-readable channel name. We don't have a use for the ID query right now, so I decided to remove it.